components/gauge/include/bq27441_defs.h

```diff
@@ -0,0 +1,117 @@
+#ifndef _BQ27441_DEFS_H_
+#define _BQ27441_DEFS_H_
+
+/**
+ * @file bq27441_defs.h
+ * @brief Definitions for the BQ27441 fuel gauge.
+ */
+
+#define BQ27441_DATA_SIZE 3 * sizeof(uint16_t) /**< Size of the buffer to hold capacity, SoC, and voltage data*/
```

Comment:
Consider renaming BQ27441_DATA_SIZE to something more descriptive, like BQ27441_BATTERY_DATA_SIZE or BQ27441_CAPACITY_SOC_VOLTAGE_SIZE, to clarify what data the buffer holds.